UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -264,6 +264,11 @@ public void AffectDamage(float damage, int damageType)
 			{
 				TakeBurnDamage(damage);
 			}
+
+			if(attackType ==  AttackType.Bomb || attackType ==  AttackType.Bullet)
+			{
+				TakeBluntDamage(damage);
```

Comment:
it already checked if the attacktype is bomb to do something, is this intended?

Reply:
I was trying to test bullet damage giving blunt trauma but trauma damage is broken so I forgot to delete that. Removing it